internal/auth/providers/google_admin.go

```diff
@@ -118,22 +137,19 @@ func (gs *GoogleAdminService) GetMembers(groupName string) ([]string, error) {
 	return members, nil
 }
 
-// GetGroups gets the groups that a user with a given email address belongs to.
-func (gs *GoogleAdminService) GetGroups(email string) ([]string, error) {
-	var groups []string
+// HasMember checks if the user has membership in the given groups
```

Comment:
Nit: `HasMember` strikes me as an odd name for a function that returns a subset of the given groups that the given email is in.  Basically, I'd expect `HasMember` to return a bool.  What about something like `CheckMembership` instead?

(Also, it might be good to expand the docstring here to more accurately describe what this function is doing w/r/t returning a subset of the given groups.)

Reply:
I wanted to reflect the underlying API endpoint we're leveraging on the google side (which is a bool endpoint, but is nice because it does nested group resolution). 

I agree this seems to imply this would be a bool by name along -- not sure what would be better.